tests/integration/security/file_mounted_certs/main_test.go

```diff
@@ -0,0 +1,206 @@
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	inst istio.Instance
+)
+
+const (
+	ProxyMetadataJson = `
+	{
+      "JWT_POLICY": "third-party-jwt",
+      "PILOT_CERT_PROVIDER": "mycopki",
+      "FILE_MOUNTED_CERTS": "true",
+      "ISTIO_META_TLS_CLIENT_CERT_CHAIN": "/client-certs/cert.pem",
+      "ISTIO_META_TLS_CLIENT_KEY": "/client-certs/key.pem",
+      "ISTIO_META_TLS_CLIENT_ROOT_CERT": "/client-certs/ca.pem",
+      "ISTIO_META_TLS_SERVER_CERT_CHAIN": "/server-certs/cert.pem",
+      "ISTIO_META_TLS_SERVER_KEY": "/server-certs/key.pem",
+      "ISTIO_META_TLS_SERVER_ROOT_CERT": "/server-certs/ca.pem",
+      "ISTIO_META_PROXY_XDS_VIA_AGENT": "false",
+      "ISTIO_METAJSON_METRICS_INCLUSIONS": '{\"sidecar.istio.io/statsInclusionPrefixes\": \"access_log_file,cluster,cluster_manager,control_plane,http,http2,http_mixer_filter,listener,listener_manager,redis,runtime,server,stats,tcp,tcp_mixer_filter,tracing\"}',
+      "ISTIO_META_DNS_CAPTURE": "false",
+	}
+`
+)
+
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite(m).
+		Label(label.CustomSetup).
+
+		// SDS requires Kubernetes 1.13
+		RequireEnvironmentVersion("1.13").
+		Label("CustomSetup").
+		Setup(istio.Setup(&inst, setupConfig, CreateCustomIstiodSecret)).
+		Run()
+}
+
+
+func setupConfig(_ resource.Context, cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+
+	cfg.ControlPlaneValues = `
+components:
+  egressGateways:
+  - enabled: false
+    name: istio-egressgateway
+  ingressGateways:
+  - enabled: false
+    name: istio-ingressgateway
+  pilot:
+    enabled: true
+    k8s:
+      overlays:
+        - kind: Deployment
+          name: istiod
+          patches:
+            - path: spec.template.spec.containers.[name:discovery].args
+              value: [
+                "discovery",
+                "--monitoringAddr=:15014",
+                "--log_output_level=default:info",
+                "--domain",
+                "cluster.local",
+                "--trust-domain=cluster.local",
+                "--keepaliveMaxServerConnectionAge",
+                "30m",
+                "--resync",
+                "5m",
+                "--caCertFile",
+                "/server-certs/ca.pem",
+                "--tlsCertFile",
+                "/server-certs/cert.pem",
+                "--tlsKeyFile",
+                "/server-certs/key.pem"
+              ]
+            - path: spec.template.spec.volumes[8]
+              value: |-
+                name: server-certs
+                secret:
+                  secretName: test-istiod-server-cred
+                  defaultMode: 420
+            - path: spec.template.spec.containers[name:discovery].volumeMounts[7]
+              value: |-
+                name: server-certs
+                mountPath: /server-certs
+
+meshConfig:
+  defaultConfig:
+    controlPlaneAuthPolicy: "NONE"
+    discoveryAddress: istiod.istio-system:15010
```

Comment:
why are we using plaintext?

Reply:
@howardjohn , the intent was to test mtls between the client and server and simplify everything else by using plaintext with control plane.
Let me change this.

Reply:
updated the PR to use mTLS with istiod